src/components/BottomSheet/components/BottomSheetContainer.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {memo, useMemo} from 'react'
+import type {ViewProps} from 'react-native'
+import {StyleProp, StyleSheet, ViewStyle} from 'react-native'
+import {GestureHandlerRootView} from 'react-native-gesture-handler'
+import Animated, {useAnimatedProps} from 'react-native-reanimated'
+import {useBottomSheet} from '../hooks'
+
+export interface BottomSheetContainerProps {
+  style?: StyleProp<ViewStyle>
+  children?: React.ReactNode
+}
+
+const BottomSheetContainerComponent: React.FC<BottomSheetContainerProps> = ({style, children}) => {
+  const {animatedIsVisible} = useBottomSheet()
+
+  const animatedProps = useAnimatedProps(
+    () => ({
+      pointerEvents: animatedIsVisible.value ? 'auto' : ('none' as ViewProps['pointerEvents']),
+    }),
+    [],
+  )
+
+  const containerStyle = useMemo<StyleProp<ViewStyle>>(
+    () => [
```

Comment:
You should use `StyleSheet.flatten` here

Reply:
updated